yt/yt/tests/integration/master/test_master_transactions.py

```diff
@@ -480,6 +480,23 @@ def test_owner(self):
         tx = start_transaction(authenticated_user="u")
         assert get("#{0}/@owner".format(tx)) == "u"
 
+    @authors("faucct")
+    def test_not_owner(self):
+        create_user("u")
+        set("//sys/schemas/transaction/@acl", [*get("//sys/schemas/transaction/@acl"), {
+            "action": "deny",
+            "subjects": ["u"],
+            "permissions": ["write"],
+            "inheritance_mode": "object_and_descendants",
+        }])
+        tx = start_transaction()
+        create("map_node", "//tmp/wut", tx=tx)
+        with pytest.raises(YtError):
```

Comment:
I'd suggest using a context manager raises_yt_error and specify particular AuthorizationError error code, similar to how it is done here
https://github.com/ytsaurus/ytsaurus/blob/180ae01baadd781a680721ef645ed4724ab79a86/yt/yt/tests/integration/queues/test_producer.py#L226